changelog/unreleased/9196

```diff
@@ -0,0 +1,6 @@
+Bugfix: Always restart OAuth2 on error
+
+We now always restart the OAuth2 process once we got a result.
```

Comment:
"We now always reset the OAuth2 status before processing the result of an authentication request."

It took me a bit to understand what the intention of this change was ;-)

Reply:
Its a bit late now after the release :P

How ever you can still provide a pull request for the change log item